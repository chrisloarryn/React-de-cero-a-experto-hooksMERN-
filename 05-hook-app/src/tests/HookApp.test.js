import React from 'react'
import { shallow } from 'enzyme/build'
import { HookApp } from 'HookApp'

describe('Tests in <HookApp />', () => {
  test('should be rendered correctly', () => {
    const wrapper = shallow(<HookApp />)
    expect(wrapper).toMatchSnapshot()
  })
})
