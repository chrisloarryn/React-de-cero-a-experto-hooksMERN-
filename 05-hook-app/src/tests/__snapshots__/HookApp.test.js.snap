// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests in <HookApp /> should be rendered correctly 1`] = `
<div>
  <h1>
     Hello World 
  </h1>
</div>
`;
